pkg/api/controller_test.go

```diff
@@ -1285,3 +1285,95 @@ func TestController_SetupLakeFSHandler(t *testing.T) {
 		})
 	}
 }
+
+func TestController_ListRepositoryRuns(t *testing.T) {
+	clt, _ := setupClientWithAdmin(t, "")
+	ctx := context.Background()
+	httpServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer httpServer.Close()
+	// create repository
+	resp, err := clt.CreateRepositoryWithResponse(ctx, &api.CreateRepositoryParams{}, api.CreateRepositoryJSONRequestBody{
+		DefaultBranch:    api.StringPtr("master"),
+		Name:             "repo9",
+		StorageNamespace: "mem://repo9",
+	})
+	verifyResponseOK(t, resp, err)
+	// upload action for pre-commit
+	actionContent := `--- 
```

Comment:
This is quite hard to read.  Can we e.g. indent?

I tried with a random Python YAML parser and with The YPaste Official YAML cgi-bin, and I think that this works (http://ben-kiki.org/ypaste/data/2571/index.html):
```yaml
---
   a:
     - 2
     - 3
   b: yes
   c: no
```

So you might be able to get by with indenting all of the lines.

Reply:
checking if it is possible or an alternative

Reply:
cloud not indent so I've extract it into a template variable - let me know if it is more readable now.